src/charm.py

```diff
@@ -145,7 +147,7 @@ def _setup_proxy_pebble_service(self):
         except ChangeError as e:
             LOGGER.error(f"Error while replanning proxy container: {e}")
 
-    def _reconcile(self, _event: ops.ConfigChangedEvent):
+    def _reconcile(self, event: ops.ConfigChangedEvent):
```

Comment:
Lol why?

Reply:
I don't even remember :laughing:.  I cherry picked these out of another PR and maybe this came from there?  idk haha but I'll revert it

Reply:
```suggestion
    def _reconcile(self, _event: ops.ConfigChangedEvent):
```